Versionize/Config/ConfigProvider.cs

```diff
@@ -92,4 +94,25 @@ private static bool MergeBool(bool overridingValue, bool? optionalValue)
     {
         return overridingValue ? overridingValue : (optionalValue ?? false);
     }
+
+    private static void ValidateChangelogPaths(FileConfig fileConfig, string cwd)
+    {
+        if (fileConfig is null || fileConfig?.Projects is null)
+        {
+            return;
+        }
+
+        var changelogPaths = new Dictionary<string, string>();
```

Comment:
Will you change this to a `HashSet<string>`? Since we only care about the paths.

Reply:
I've made the change. Initially, I considered it, but then I thought about including the project name in the error message. Perhaps in the future, it could be more specific to the user.

Reply:
Good point. Yeah, we can adjust later if/when we make the message more specific.